.gitignore

```diff
@@ -31,3 +31,74 @@ build/
 
 ### VS Code ###
 .vscode/
+
+# Created by https://www.toptal.com/developers/gitignore/api/eclipse
+# Edit at https://www.toptal.com/developers/gitignore?templates=eclipse
+
+### Eclipse ###
+.metadata
+bin/
+tmp/
+*.tmp
+*.bak
+*.swp
+*~.nib
+local.properties
+.settings/
+.loadpath
+.recommenders
```

Comment:
I updated the .gitignore file with items specific to Eclipse using gitignore.io. There are some files you could come across during development that are specific to each developer's system, so you don't really want to push those to Github. I believe that includes maven-wrapper and mvnw files, which you guys have in your project, but they didn't seem to effect how the project runs so I left those alone.

Professor Brandan Jones has a video talking about .gitignore for more info:  https://www.youtube.com/watch?v=tBGjlXRWmik